.github/workflows/scalar-functional-tests.yml

```diff
@@ -1,9 +1,10 @@
 name: Scalar Functional Tests
 
 env:
-  SCALAR_REPOSITORY: dscho/scalar
-  SCALAR_REF: vfs-2.32.0
+  SCALAR_REPOSITORY: derrickstolee/scalar
+  SCALAR_REF: test-scalar-in-c
   DEBUG_WITH_TMATE: false
+  SCALAR_TEST_SKIP_VSTS_INFO: true
```

Comment:
I'll need to investigate the need for this. I would like to have coverage of the `/vsts/info` endpoint, if possible.

Reply:
It's not really possible because that endpoint is _authenticated_. Scalar.NET completely side-steps this by ignoring the need for authentication here and skipping it.

Reply:
Even on a public project? Dang. OK.